cmd/stack.go

```diff
@@ -20,19 +26,34 @@ func setupStackCommand() *cobra.Command {
 				return cobraext.FlagParsingError(err, cobraext.DaemonModeFlagName)
 			}
 
+			services, err := cmd.Flags().GetStringSlice(cobraext.StackServicesFlagName)
```

Comment:
Ah, didn't know about `GetStringSlice` - nice! Mind converting the parsing of the `--dataset` flag in the `test` command to use this same mechanism?

Reply:
Fixed!